pkg/master/reconcilers/lease.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconcilers
+
+/*
+Original Source:
```

Comment:
I don't think this comment helps much. If people are `git blaming`, they'll find the diff where it was created / moved anyways.

Reply:
It's from OpenShift, not Kubernetes though.

Reply:
Oops, missed that! Yeah, I'm happy with leaving it then.